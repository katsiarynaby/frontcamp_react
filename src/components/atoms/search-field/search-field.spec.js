import React from 'react';
import { shallow } from 'enzyme';
import shallowToJson from 'enzyme-to-json';

import SearchField from './index';

describe('SearchField', () => {
    it("SearchField should render correctly", () => {
        const wrapper = shallow(<SearchField getValue='getValue' />);

        expect(shallowToJson(wrapper)).toMatchSnapshot();
    });
});
