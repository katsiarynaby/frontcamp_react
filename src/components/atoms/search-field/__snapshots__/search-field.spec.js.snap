// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchField SearchField should render correctly 1`] = `
<input
  className="search-field"
  onChange={[Function]}
  placeholder="SEARCH"
  type="search"
/>
`;
